deploy/docker/x86_64/Dockerfile

```
@@ -13,17 +13,38 @@
 # limitations under the License.
 
 # Modified from https://github.com/rootfs/nfs-ganesha-docker by Huamin Chen
-FROM fedora:30 AS build
+ARG FEDORA_VERSION=34
+
+FROM fedora:${FEDORA_VERSION} AS build
 
 # Build ganesha from source, install it to /usr/local and a use multi stage build to have a smaller image
 # Set NFS_V4_RECOV_ROOT to /export
 
-RUN dnf install -y tar gcc cmake-3.14.2-1.fc30 autoconf libtool bison flex make gcc-c++ krb5-devel dbus-devel jemalloc-devel libnfsidmap-devel libnsl2-devel userspace-rcu-devel patch libblkid-devel
-RUN curl -L https://github.com/nfs-ganesha/nfs-ganesha/archive/V2.8.2.tar.gz | tar zx \
-	  && curl -L https://github.com/nfs-ganesha/ntirpc/archive/v1.8.0.tar.gz | tar zx \
-	  && rm -r nfs-ganesha-2.8.2/src/libntirpc \
-	  && mv ntirpc-1.8.0 nfs-ganesha-2.8.2/src/libntirpc
-WORKDIR /nfs-ganesha-2.8.2
+# Install dependencies on separated lines to be easier to track changes using git blame
+RUN dnf install -y \
+	autoconf \
+	bison \
+	cmake \
+	dbus-devel \
+	flex \
+	tar \
+	gcc \
+	gcc-c++ \
+	git \
+	jemalloc-devel \
+	krb5-devel \
+	libblkid-devel \
+	libnfsidmap-devel \
+	libnsl2-devel \
+	libtool \
+	make \
+	patch \
+	userspace-rcu-devel
+
+# Clone specific version of ganesha
+ARG GANESHA_VERSION=V3.4
```

While updating this as part of this PR anyhow.

https://github.com/nfs-ganesha/nfs-ganesha/tree/V3.5

```suggestion
ARG GANESHA_VERSION=V3.5
```

---

Oh, I see now that V3.4 was used in the repo root Dockerfile